Sources/Parley/Data/Models/Message.swift

```diff
@@ -63,7 +63,7 @@ public class Message: Codable, Equatable {
     }
 
     internal var hasButtons: Bool {
-        (buttons?.count ?? 0) > 0
+        buttons?.isEmpty == false
```

Comment:
Revert change: When `buttons` is `null` this will return true in the updated way